### DISCUSSION TITLE
Support for group DReps

### DISCUSSION THREAD

**Discussion by l-br1** [IntersectMBO/govtool#1761] [Ideas]
This is to discuss what are the needs of group DReps and what are the options to handle registration and delegation in govtool

https://github.com/IntersectMBO/govtool/issues/1767

**Comment by l-br1** [comment]
@Ryun1

**Comment by phillewis** [comment]
Might be better to have this customer-driven rather than developer-driven, to avoid building solutions for presumed problems #YAGNI

There are many ways that group DReps could choose to operate, e.g.:
-  Multi-sig with doxxed key-holders
- Single key with off-chain private or public governance
- Single key with on-chain governance using another token (e.g. as Minswap did for Catalyst)

**Reply by QSchlegel** [reply]
Or mesh Multisig 😉

**Reply by l-br1** [reply]
we do then need to check which type of Group DRep would used Govtool OR simply choose the option that would be easier for a non-tech user (as the goal is also to make it easy for group of people to become group DReps)

**Reply by l-br1** [reply]
also this might be a two phases:
- Support for existing group DReps (so they don't need to use the CLi) 
- create group DReps

**Reply by l-br1** [reply]
this in general could support expansion of more DReps as groups of people with common ideas

**Reply by QSchlegel** [reply]
multisig.meshjs.dev

**Comment by m-i-k-e** [comment]
What is the definition of a 'group DRep'? Are there any user stories that can be written at this point?

**Comment by m-i-k-e** [comment]
We should see what users want by talking to them first.

**Comment by QSchlegel** [comment]
Well there are "group" dreps, which are script dreps. Using multisig wallets.
Why not support them as they already exist.
Also not to difficult as ekklesia already supports that.
